code/persistentstorage/Databases.md

```diff
@@ -0,0 +1,108 @@
+# Databases
+
+This document decribes the following:
+- [The various databases used for each component in the Storj network.](#Storj-components-with-databases)
+- [How to view the contents of the various databases.](#View-the-Contents-of-a-Database)
+- [Steps on how to modify existing databases.](#Steps-to-Modify-Existing-Databases)
+
+
+## Storj Components With Databases
+
+Each component of the Storj network (i.e. satellites, storage nodes, uplinks) has different persistent data stores.
+
+Satellites have three databases:
+1. SatelliteDB, also called the master database.
+2. PointerDB.
+3. Kademlia Routing Table.
+
+Storage Nodes have two databases:
+1. Piecestore database.
+2. Kademlia Routing Table.
```

Comment:
Same here, there's one master database and then it contains multiple other databases.

Storage Node has this breakdown (with hammer branch)

1. used serials - tracking which serials have been used
2. certificates - tracking uplink certificates
3. pieceinfo - tracking what pieces are currently stored
4. bandwidth usage - tracking how much data we have transferred
5. unsent orders, order archive - tracking what needs to be sent to the satellite
6. piece storage - saving the actual pieces
7. kademlia routing table - storing the data